app/src/main/java/com/coveros/coverosmobileapp/website/MainActivity.java

```diff
@@ -187,11 +214,26 @@ public void onClick(DialogInterface dialogs, int which){
          */
         @Override
         public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
-            if (position == 0) {
-                startActivity(new Intent(getApplicationContext(), MainActivity.class));
-            } else {
+            String website;
+            String[] stringsToAddLast = {"","#projects","products/secure-ci/","products/selenified/","#services", "services/agile-development/", "services/agile-testing/",
```

Comment:
Since all of this is duplicated, refactor this so the logic exists in a single place/class